test/TEST-24-CRYPTSETUP-KEYDEV/test.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+set -e
+TEST_DESCRIPTION="unlocking of encrypted disk using password on the different block device"
```

Comment:
As discussed on telegram, we assume that tests pass locally without the binaries from git being installed to the system, and especially without them being installed into the initramfs.

Also, I don't think the test is working. Without an updated initramfs, it just waits for the password, and if I just press `<enter>` a few times, the test continues, an reports `TEST RUN: unlocking of encrypted disk using password on the different block device [OK]` at the end. I would expect a failure, since the password wasn't entered correctly.